prod/commonvars.hcl

```diff
@@ -10,4 +8,9 @@ locals {
     "51.144.56.160/28",
     "51.144.56.176/28",
   ]
+
+  cet_time_zone_win = "Central Europe Standard Time"
```

Comment:
`win` is for windows?

Reply:
yes, unfortunately values can be different between service plan based on linux rather than window.